pytorch_lightning/overrides/torch_distributed.py

```diff
@@ -0,0 +1,188 @@
+import io
+import logging
+import os
+import pickle
+
+import torch
+from torch._C._distributed_c10d import ProcessGroup
+
+from pytorch_lightning.utilities import _HPU_AVAILABLE
+from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_8
+
+_pickler = pickle.Pickler
+_unpickler = pickle.Unpickler
+
+
+logger = logging.getLogger(__name__)
+
+if torch.distributed.is_available():
```

Comment:
PL 1.6 will drop PyTorch 1.7 support

So you can assume PyTorch 1.8 will be the minimum version